code/src/components/EndScreen.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
```

Comment:
Separating the end screen into a different component was a good idea in this case. You have several other components, so this approach keeps the code clean and understandable.